### ISSUE TITLE
systemd-shutdown gets constipated (Bugzilla Bug 2656)

### ISSUE THREAD

**Issue by tpgxyz** [OpenMandrivaAssociation/test2#2656]
This issue was created automatically with bugzilla2github
# Bugzilla Bug 2656
Date: 2020-11-10 02:39:56 +0000
From: @benbullard79
To:   @berolinux
CC:   @benbullard79, bugs@openmandriva.org, nix.or.die@gmail.com, nobodydead@gmail.com, @tpgxyz

Depends on:   #2660
Last updated: 2021-02-03 15:02:46 +0000

## Comment 20434
Date: 2020-11-10 02:39:56 +0000
From: @benbullard79

This happens intermittently and I don't know any good way to reproduce this. The people most likely to see this would be people that regularly reboot their box. So perhaps testers and multi-booters like me would see this as a bigger problem where as far as I know a lot of developers don't reboot as often

Also I have not yet provided any logs because I'm not sure how to document a problem with shutdown. So if anyone knows please inform me otherwise I'll be looking and will provide logs when I learn how to do so for this type of problem.

Basically maybe 10% of the reboots I do computer gets stuck and if I access output with the ESC key I see something like this:

systemd-shutdown[1]: Waiting for process: systemd, dbus-broker, dbus-broker-lau, kactivitymanager, kded5

These 4 seem always present: systemd, dbus-broker, dbus-broker-lau, kactivitymanager

otherwise I have seen other things in the list.

One point of interest is that my normal procedure for shutdown is to close all open applications, logout, then either restart or shutdown.

Attached is the journal as user of most recent time this happened. It will show:

Nov 09 20:17:45 ben-Rzyen5 sddm[886]: Error from greeter session: "Process crashed"
Nov 09 20:17:45 ben-Rzyen5 sddm[886]: Auth: sddm-helper crashed (exit code 15)
Nov 09 20:17:45 ben-Rzyen5 sddm[886]: Error from greeter session: "Process crashed"
Nov 09 20:17:45 ben-Rzyen5 sddm[886]: Auth: sddm-helper exited with 15

don't know if that is relevant. Then at the end:

Nov 09 20:17:45 ben-Rzyen5 systemd[1]: Started Show Plymouth Reboot Screen.
Nov 09 20:17:59 ben-Rzyen5 systemd[1]: Received SIGINT.
Nov 09 20:17:59 ben-Rzyen5 systemd[1]: Received SIGINT.
Nov 09 20:17:59 ben-Rzyen5 systemd[1]: Received SIGINT.
Nov 09 20:18:00 ben-Rzyen5 systemd[1]: Received SIGINT.
Nov 09 20:18:00 ben-Rzyen5 systemd[1]: Received SIGINT.
Nov 09 20:18:00 ben-Rzyen5 systemd[1]: Received SIGINT.
Nov 09 20:18:00 ben-Rzyen5 systemd[1]: Received SIGINT.
Nov 09 20:18:00 ben-Rzyen5 systemd[1]: Received SIGINT.
Nov 09 20:18:00 ben-Rzyen5 systemd[1]: Forcibly rebooting: Ctrl-Alt-Del was pressed more than 7 times within 2s
Nov 09 20:18:00 ben-Rzyen5 systemd[1]: Shutting down.
Nov 09 20:18:00 ben-Rzyen5 systemd-shutdown[1]: Syncing filesystems and block devices.
Nov 09 20:18:00 ben-Rzyen5 systemd-shutdown[1]: Sending SIGTERM to remaining processes...
Nov 09 20:18:00 ben-Rzyen5 systemd-journald[566]: Journal stopped

Not sure but it *seems* like using Ctrl-Alt-Del makes the process go faster.

## Comment 20435
Date: 2020-11-10 02:41:14 +0000
From: @benbullard79

Created attachment 1929
journalctl -b -1

Bugzilla is ...

> Attached file: journal.txt (text/plain, 457925 bytes)
> Description:   journalctl -b -1

## Comment 20439
Date: 2020-11-10 02:57:56 +0000
From: @benbullard79

*** Bug #2655 has been marked as a duplicate of this bug. ***

## Comment 20494
Date: 2020-11-29 17:32:34 +0000
From: @benbullard79

I am Confirming this in hopes of getting some action on this.

Also seems I have found a way to reproduce.

Logout and try to log back in. If your computer is like mine login will get stuck on KSplash screen and you will have to Ctl>Alt>Bksp and then restart. When I do all of that I get this bug every time.

## Comment 20499
Date: 2020-11-30 11:04:16 +0000
From: @tpgxyz

Sounds like related to https://issues.openmandriva.org/show_bug.cgi?id=2660

## Comment 20581
Date: 2021-02-03 13:47:34 +0000
From: @tpgxyz

Does this still occur ?

## Comment 20584
Date: 2021-02-03 15:02:46 +0000
From: @benbullard79

I have not noticed this recently. Looked at journal for some recent boots and the shutdown happens in normal time frame.

So marking RESOLVED/FIXED.
